content/blog/automation-api-python/index.md

```diff
@@ -0,0 +1,161 @@
+    ...
+```
+
+Check out the full `Pulumi over HTTP` example in [Python](https://github.com/pulumi/automation-api-examples/tree/main/python/pulumi_over_http).
+
+### Complex Workflow Orchestration
+
+The Automation API executes just like any other code in your application, meaning that a single process can deploy infrastructure and manage related activities such as database migrations and blue/green deployments.
+
+In this example, we define an AWS RDS database:
+
+```python
+cluster_instance = aws.rds.ClusterInstance(
+    "db_instance",
+    cluster_identifier=cluster.cluster_identifier,
+    instance_class=aws.rds.InstanceType.T3_SMALL,
+    engine=aws.rds.EngineType.AURORA_MYSQL,
+    engine_version="5.7.mysql_aurora.2.03.2",
+    publicly_accessible=True,
+    db_subnet_group_name=subnet_group.name
+)
+
+pulumi.export("host", cluster.endpoint)
+pulumi.export("db_name", db_name)
+pulumi.export("db_user", db_user)
+pulumi.export("db_pass", db_pass)
+```
+
+With Automation API, we can perform database migrations and preload rows in the same program:
+
+```python
+# create (or select if one already exists) a stack that uses our inline program
+stack = auto.create_or_select_stack(stack_name=stack_name,
+                                    project_name=project_name,
+                                    program=pulumi_program)
+
+up_res = stack.up(on_output=print)
+print(f"db host url: {up_res.outputs['host'].value}")
+
+print("configuring db...")
+with connect(
+        host=up_res.outputs['host'].value,
+        user=up_res.outputs['db_user'].value,
+        password=up_res.outputs['db_pass'].value,
+        database=up_res.outputs['db_name'].value) as connection:
+    print("db configured!")
+
+    # make sure the table exists
+    print("creating table...")
+    create_table_query = """CREATE TABLE IF NOT EXISTS hello_pulumi(
+        id int(9) NOT NULL PRIMARY KEY,
+        color varchar(14) NOT NULL);
+        """
+    with connection.cursor() as cursor:
+        cursor.execute(create_table_query)
+        connection.commit()
+
+```
+
+See the full database deployment and migration Automation API program in [Python](https://github.com/pulumi/automation-api-examples/tree/main/python/database_migration).
+
+## Automation API Resources
+
+To get started with Automation API checkout these links:
+
+- [Python documentation]({{< relref "/docs/reference/pkg/python/pulumi#module-pulumi.x.automation" >}})
+- [Automation API examples](https://github.com/pulumi/automation-api-examples#python-examples)
+- [Generating S3 static websites using Pulumi's Automation API in Python](https://www.youtube.com/watch?v=8XFjqzX9ZK4&t=64s&ab_channel=PulumiTV)
+
+Automation API is still in alpha. We maintain a list of [known issues](https://github.com/pulumi/pulumi/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fautomation-api). Please [file more](https://github.com/pulumi/pulumi/issues/new?assignees=&labels=needs-triage&template=bug_report.md&title=) as you find them!
```

Comment:
what do you define as an "alpha" feature?

Reply:
I don't define the release version, it's how it was introduced. Alex, should set the criteria for release versions.